### PR TITLE
Add link to Discord server

### PR DESCRIPTION
**Pull request by SiarheiFedartsou**
# Issue

I created Discord server for us, not sure if people will actually use it, but probably it will help building community and will allow to have ways for more interactive communications. So let's try. :) 

## Tasklist

 - [ ] CHANGELOG.md entry ([How to write a changelog entry](http://keepachangelog.com/en/1.0.0/#how))
 - [ ] update relevant [Wiki pages](https://github.com/Project-OSRM/osrm-backend/wiki)
 - [ ] add tests (see [testing documentation](https://github.com/Project-OSRM/osrm-backend/blob/master/docs/testing.md))
 - [ ] review
 - [ ] adjust for comments
 - [ ] cherry pick to release branch

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +32,7 @@ Related [Project-OSRM](https://github.com/Project-OSRM) repositories:
 
 ## Contact
 
+- Discord: [https://discord.gg/es9CdcCXcb](join)
 - IRC: `irc.oftc.net`, channel: `#osrm` ([Webchat](https://webchat.oftc.net))
```
Comment (SiarheiFedartsou) [inline README.md:36]: Do we still need links to IRC and Mailinglist here? They seem to be completely dead...

File: README.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
 ## Open Source Routing Machine
 
-| Linux / macOS / Windows | Code Coverage |
-| ----------------------- | ------------- |
-| [![osrm-backend CI](https://github.com/Project-OSRM/osrm-backend/actions/workflows/osrm-backend.yml/badge.svg)](https://github.com/Project-OSRM/osrm-backend/actions/workflows/osrm-backend.yml) | [![Codecov](https://codecov.io/gh/Project-OSRM/osrm-backend/branch/master/graph/badge.svg)](https://codecov.io/gh/Project-OSRM/osrm-backend) |
+
+[![osrm-backend CI](https://github.com/Project-OSRM/osrm-backend/actions/workflows/osrm-backend.yml/badge.svg)](https://github.com/Project-OSRM/osrm-backend/actions/workflows/osrm-backend.yml) [![Codecov](https://codecov.io/gh/Project-OSRM/osrm-backend/branch/master/graph/badge.svg)](https://codecov.io/gh/Project-OSRM/osrm-backend) [![Discord](https://img.shields.io/discord/1034487840219860992)](https://discord.gg/es9CdcCXcb)
```
Comment (SiarheiFedartsou) [inline README.md:4]: Screenshot 2022-10-25 at 20 21 03

File: README.md (hunk 1 above)
Comment (mjjbell) [reply README.md:36]: There are some useful historical discussions in the mailing list, so good to keep as a reference. Not so sure about IRC.

**Review by mjjbell — APPROVED** [review]

**Comment by jcoupey** [comment]
> Not so sure about IRC.

FWIW, there are a few users still hanging around over there. I would not say the channel is very active but we do get occasional exchanges, mostly whenever someone pops in with a question.

Having an active communication channel is great, but having the community split across different channels may be a shame. I have no strong opinion on which is the best option, but I do get that IRC can sound a bit old-fashioned nowadays. And it has the major drawback that you can't wave to say hi using fancy animal animations. ;-)

**Pull request merged**